src/controllers/album.js

```diff
@@ -0,0 +1,101 @@
+const getDb = require("../services/db");
+
+// creates a new album
+exports.create = async (req, res) => {
+  const db = await getDb();
+  const { name, year } = req.body;
+  const { artistId } = req.params;
+  // console.log(artistId);
```

Comment:
Console logs (that dont have an actual programmatic use such as logging debugging info like on line 18) should be removed before submitting for review